spec/requests/citizens/outgoings_summary_spec.rb

```diff
@@ -1,41 +1,50 @@
 require 'rails_helper'
 
 RSpec.describe Citizens::OutgoingsSummaryController do
-  let(:legal_aid_application) { create :legal_aid_application, :with_applicant }
+  let!(:rent_or_mortgage) { create :transaction_type, :debit, name: 'rent_or_mortgage' }
+  let!(:council_tax) { create :transaction_type, :debit, name: 'council_tax' }
```

Comment:
Just a warning that I've removed the council_tax transaction type in another PR (https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/345) which is hasn't been merged to master yet (still waiting UAT) so there will be a merge conflict on this file for one of us at some point...

Reply:
I've anonymised the transactions types - so that they are just `transaction_types` (associated with the `legal_aid_application` and `other_transaction_types` (not associated) in the request spec. Then the factory can determine which transaction types are used, and that should keep in step with the applied ones.